docs/docs/apis/openidoauth/grant-types.md

```diff
@@ -14,7 +14,7 @@ For a list of supported or unsupported `Grant Types` please have a look at the t
 | JSON Web Token (JWT) Profile                          | yes                 |
 | Refresh Token                                         | yes                 |
 | Resource Owner Password Credentials                   | no                  |
-| Security Assertion Markup Language (SAML) 2.0 Profile | no                  |
+| Security Assertion Markup Language (SAML) 2.0 Profile | yes                 |
```

Comment:
```suggestion
| Security Assertion Markup Language (SAML) 2.0 Profile | no                 |
```

Reply:
The SAML grant inside oidc/oauth is not yet supported (this would return a saml token instead of a JWT, opaque)

Reply:
ah. my bad, thanks.